hack/e2e/run-rp-and-e2e.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash -e
 ######## Helper file to run E2e either locally or using Azure DevOps Pipelines ########
 
+set -x
```

Comment:
why not and line 1?

Reply:
because the file is sourced - the shebang is irrelevant in this file.